relay-server/src/actors/outcome.rs

```diff
@@ -554,7 +554,7 @@ impl HttpOutcomeProducer {
             match compat::send(UpstreamRelay::from_registry(), SendQuery(request)).await {
                 Ok(_) => relay_log::trace!("outcome batch sent."),
                 Err(error) => {
-                    relay_log::error!("outcome batch sending failed with: {}", LogError(&error))
+                    relay_log::error!("outcome batch sending failed with: {}", error)
```

Comment:
This `LogError` was removed since the underlying `MailboxError` doesn't implement `Error` and it also doesn't contain nested sources. This can be reverted when converting `UpstreamRelay` to a service.